nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -3,43 +3,52 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-#import hashlib
-#import itertools
 import typing as tp
-import numpy as np
-import sklearn.datasets  # type:ignore
-
 from functools import partial
+import numpy as np
+import sklearn.datasets
+from sklearn.tree import DecisionTreeRegressor
+from sklearn.neural_network import MLPRegressor
 
 from nevergrad.parametrization import parameter as p
-from nevergrad.common import tools
-from nevergrad.common.typetools import ArrayLike
-from sklearn.tree import DecisionTreeRegressor  # type: ignore
-from sklearn.neural_network import MLPRegressor  # type: ignore
-
 from ..base import ExperimentFunction
-from .. import utils
-from .. import corefuncs
-
-# EF = tp.TypeVar("EF", bound="ExperimentFunction")
 
 
+# pylint: disable=too-many-instance-attributes,too-many-locals,too-many-arguments
 class MLTuning(ExperimentFunction):
     """Class for generating ML hyperparameter tuning problems.
+    We propose different possible regressors and different dimensionalities.
```

Comment:
@teytaud 
The init docstring needs to go write after the class def, not the init